### ISSUE TITLE
A potential coding bug ?

### ISSUE THREAD

**Issue by leo-frank** [dcharatan/flowmap#40]
In [pick rgb colors](https://github.com/dcharatan/flowmap/blob/8bd8d33e6f5cf7a99e4cffbd1dc5a6caa91ad229/flowmap/tracking/track_predictor_cotracker.py#L47), you directly use xy to pick colors from images. But in fact, xy are in range [0, 1], I think you should use 2*xy-1.

**Comment by dcharatan** [comment]
Yes, this is definitely a bug. Thank you for discovering this! I've pushed a commit that should fix it.
